pages/blocks/video/video.js

```diff
@@ -31,7 +32,10 @@ export default async function decorate(block) {
       wrapper.innerHTML = '<div class="video-placeholder-play"><button title="Play"></button></div>';
       wrapper.prepend(pic);
       wrapper.addEventListener('click', () => {
-        renderVideo(block, source, true);
+        requestAnimationFrame(() => {
```

Comment:
curious about the use of `requestAnimationFrame` to be honest, I've never used it, so if it's required then fair enough, but reading what it does at https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame it doesn't seem to me like it should be required in this case (but what do I know)

Reply:
Thanks I've removed... I had left that in from first solution